pkgs/by-name/ne/newsraft/package.nix

```
@@ -31,11 +30,14 @@ stdenv.mkDerivation (finalAttrs: {
     gumbo
     ncurses
     sqlite
-    yajl
   ];
 
   makeFlags = [ "PREFIX=$(out)" ];
 
+  postInstall = ''
+    install -Dm444 doc/newsraft.desktop -t $out/share/applications
+  '';
```

```suggestion
  postInstall = lib.optionalString (!stdenv.hostPlatform.isDarwin) ''
    install -Dm444 doc/newsraft.desktop -t $out/share/applications
  '';
```

Note: added Darwin only, but I'm not sure if other platforms within `lib.platforms.all` would need that exclusion too (FreeBSD, WSL)?

---

I believe FreeBSD and WSL2 can both make use of `.desktop` files, so this should be fine.